silent_ping.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0"
+  spec.add_development_dependency "bundler", ">= 1.16.1"
```

Comment:
:( so sad

Reply:
Yeah, we still need that to support 2.3.6. 😬